<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kunde Registrieren</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ressources/icons/logo_smalltrans.png</normaloff>ressources/icons/logo_smalltrans.png</iconset>
  </property>
  <widget class="QFrame" name="formLogin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="customerName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Kundename*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerCity">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCity">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerEmail">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerCity">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Stadt*</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customerPhone">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius : 3px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerTelefon">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Telefon*</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="customerBirthDate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerBirthDate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Gebutsdatum*</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="customerAnotherInfo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 3px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerAnotherInfo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Sonstige Infos</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCustomerTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Kunde Registrieren</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCustomerRegister">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe MDL2 Assets</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(252, 122, 42);
border-radius: 8px;
color:white;
}
QPushButton:hover{
background-color: rgb(0, 189, 0);
border-radius: 8px;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Registrieren</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>0</y>
      <width>731</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ressources/images/bg11.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dark_bg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>541</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(156, 139, 116,120);
border-radius: 8px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteCustomer">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnDeleteCustomer{
	font: 11pt &quot;MS Shell Dlg 2&quot;;
	color: rgb(255, 255, 255);
	border-radius:5px;
	background-color: rgb(254, 52, 1);
}
QPushButton#btnDeleteCustomer:hover{
	border: 2px solid rgb(255,255,255);

}</string>
    </property>
    <property name="text">
     <string>Löschen</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>ressources/icons/recycle-bin.png</normaloff>ressources/icons/recycle-bin.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="txtPflichtfelder">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>530</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Black</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>*Pflichtfelder</string>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>dark_bg</zorder>
   <zorder>customerName</zorder>
   <zorder>txtCustomerName</zorder>
   <zorder>customerCity</zorder>
   <zorder>txtCity</zorder>
   <zorder>customerEmail</zorder>
   <zorder>txtCustomerCity</zorder>
   <zorder>customerPhone</zorder>
   <zorder>txtCustomerTelefon</zorder>
   <zorder>customerBirthDate</zorder>
   <zorder>txtCustomerBirthDate</zorder>
   <zorder>customerAnotherInfo</zorder>
   <zorder>txtCustomerAnotherInfo</zorder>
   <zorder>txtCustomerTitle</zorder>
   <zorder>btnCustomerRegister</zorder>
   <zorder>btnDeleteCustomer</zorder>
   <zorder>txtPflichtfelder</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
